dropwizard-benchmarks/pom.xml

```diff
@@ -23,6 +23,21 @@
     <artifactId>dropwizard-benchmarks</artifactId>
     <name>Dropwizard Benchmarks</name>
 
+    <build>
+        <plugins>
```

Comment:
The nullaway configuration also allows you to exclude packages there too if we want to excludes these packages in the main pom

Reply:
Yes, it will be probably a better approach.